scalafix/output/src/main/scala/rsc/tests/BetterRscCompat.scala

```diff
@@ -263,4 +263,12 @@ object BetterRscCompat_Test {
   object NestedTuples {
     val foo: ((String, Int), Double) = (("1", 1), 1.0)
   }
+
+  object PolymorphicGrandparent {
+    class C[A]
```

Comment:
We probably want to have test cases for `abstract class` instead of `class` or `trait`.